### PR TITLE
Setup Webpack config

### PR DESCRIPTION
**Pull request by Karyum**
#118

### CODE REVIEW

File: webpack.dev.config.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      }
+    ]
+  },
+  resolve: {
+    extensions: ['.js']
+  },
+  watch: true,
+  plugins: [
+    new webpack.DefinePlugin({
+      'process.env': {
```
Comment (finnhodgkin) [inline webpack.dev.config.js]: This shouldn't be called `process.env`. It's not coming from your `.env`. Pretty confusing for people coming to your project fresh. I'd rename it `webpack.env` or something :+1:

File: webpack.prod.config.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    extensions: ['.js']
+  },
+  plugins: [
+    new webpack.optimize.UglifyJsPlugin({
+      compress: {
+        warnings: true
+      }
+    }),
+    new webpack.DefinePlugin({
+      'process.env': {
```
Comment (finnhodgkin) [inline webpack.prod.config.js]: Same as above.

File: public/js/helpers/get_startTime.js [hunk 3]
```diff
@@ -1,25 +1,26 @@
...
-      // calendar next and prev month buttons
-      document.getElementById('next-button').onclick = function () {
-        calendar.nextMonth(data);
-        addDateEventListeners();
-      };
-      document.getElementById('prev-button').onclick = function () {
-        calendar.previousMonth(data);
-        addDateEventListeners();
-      };
+  axios.get(`${process.env.URI}/events`).then(function (res) {
```
Comment (finnhodgkin) [inline public/js/helpers/get_startTime.js]: Are you going to convert to es6 now you're webpacking?

Comment (finnhodgkin) [reply public/js/helpers/get_startTime.js]: Probably a different PR

Comment (Karyum) [reply public/js/helpers/get_startTime.js]: yup this PR is just one part of the main issue #106

**Pull request merged**